src/main/java/org/skriptlang/skript/bukkit/registration/BukkitInfos.java

```diff
@@ -0,0 +1,318 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.bukkit.registration;
+
+import ch.njol.skript.lang.SkriptEvent;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.registration.BukkitInfosImpl.EventImpl;
+import org.skriptlang.skript.lang.entry.EntryValidator;
+import org.skriptlang.skript.registration.SyntaxInfo;
+
+import java.util.Collection;
+
+/**
+ * A class containing the interfaces representing Bukkit-specific SyntaxInfo implementations.
+ */
+@ApiStatus.Experimental
+public final class BukkitInfos {
```

Comment:
I'm not a huge fan of this class name, do you think we could call it something a bit nicer?

Reply:
BukkitSyntaxInfos? I'm not too sure what else might work☹️